src/components/basic/lists/AlternateListItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const AlternateListItem = props => (
+  <li>
+    <a href="#">
+      {props.children}
+      <i className="fa fa-angle-right dg_i-float-right" aria-hidden="true"></i>
```

Comment:
I think in this case, we probably want to achieve the arrow with css, since it is just decoration.

Reply:
Working on this now. Will commit when finished.

Reply:
Submitted via the latest commit.